app/Http/Livewire/Settings/Update.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Http\Livewire\Settings;
+
+use App\Helpers\GitHandler;
+use Livewire\Component;
+use Illuminate\Support\Facades\Artisan;
+
+class Update extends Component
+{
+    protected $listeners = [
+        'updateProject'
+    ];
+
+    public $message = "";
+
+    public function updateProject()
+    {
+      
+        $gitHandler = new GitHandler();
+        $updated = $gitHandler->fetchAndPull();
+        // $updatedPush = $gitHandler->pushChanges();
+    
+        if ($updated) {
+            return "Project featched";
+        } elseif ($updatedPush){
+            return "updating project";
+    } else {
+            return "Error updating project";
+        }
+
```

Comment:
```suggestion
        if ($updated) {
            return 'Project fetched';
        } elseif ($updatedPush){
            return 'updating project';
        } 
        
        return 'Error updating project';
```
1. Use `'string'` for string.
2. Fixed a spelling error.
3. Removed unnecessary `else`